sunpy/coordinates/tests/test_utils.py

```diff
@@ -32,8 +32,20 @@ def test_great_arc_calculable(start, end):
                  observer=frames.HeliographicStonyhurst(0*u.deg, 0*u.deg, 1*u.AU))
     gc = GreatArc(c, d)
 
-    assert gc.start == c
-    assert gc.end == d
+    assert gc.start.x == c.transform_to(frames.Heliocentric).x
```

Comment:
Here and later, perform the transformation once for each set of components rather than once for each component.